### ISSUE TITLE
Training loss doesn't converge 

### ISSUE THREAD

**Issue by wangwen39** [mrsalehi/TGNS#2]
Thanks for sharing your codes. 
I used the code you provide with the original experimental settings on ActivityNet dataset, but the training loss doesn't converge, and I tried a smaller learning rate, but it still appeared this issue. Have you ever met this problem? Could you give me some advice?
